common/src/main/java/com/linecorp/centraldogma/common/Change.java

```diff
@@ -256,6 +287,63 @@
         return new DefaultChange<>(path, ChangeType.APPLY_JSON_PATCH, jsonPatchNode);
     }
 
+    /**
+     * Returns a newly-created {@link Change} whose type is {@link ChangeType#APPLY_YAML_PATCH}.
+     *
+     * @param path the path of the file
+     * @param oldYamlText the old content of the file
+     * @param newYamlText the new content of the file
+     *
+     * @throws ChangeFormatException if the specified {@code oldYamlText} or {@code newYamlText} is
+     *                               not a valid YAML
+     */
+    static Change<JsonNode> ofYamlPatch(String path, @Nullable String oldYamlText, String newYamlText) {
+        requireNonNull(newYamlText, "newYamlText");
+
+        final JsonNode oldYamlNode;
+        final JsonNode newYamlNode;
+        try {
+            oldYamlNode = oldYamlText == null ? JacksonYaml.nullNode
+                                              : JacksonYaml.readTree(oldYamlText);
```

Comment:
How about making `JacksonYaml.readTree()` return `JacksonYaml.nullNode` if `oldYamlText` is null?

Reply:
Oh, I copied it from `ofJsonPatch`. I'll apply it after modify `Jackson.readTree()` receive `EntryType`.